test/com/facebook/buck/ide/intellij/DefaultIjLibraryFactoryTest.java

```diff
@@ -49,13 +51,15 @@
   private IjLibrary guavaLibrary;
   private TargetNode<?, ?> androidSupport;
   private FakeSourcePath androidSupportBinaryPath;
+  private FakeSourcePath androidSupportResClassPath;
```

Comment:
this should've been used on new L139. in the added test condition. will fix my copy/paste error